x/crosschain/keeper/cctx_orchestrator_validate_inbound.go

```diff
@@ -20,9 +21,8 @@ func (k Keeper) ValidateInbound(
 	if !tssFound {
 		return nil, types.ErrCannotFindTSSKeys
 	}
-	err := k.CheckIfTSSMigrationTransfer(ctx, msg)
-	if err != nil {
-		return nil, err
+	if err := k.CheckIfTSSMigrationTransfer(ctx, msg); err != nil {
+		return nil, errors.Wrap(err, "tss migration transfer check failed")
```

Comment:
I personally prefer the error check on a separate line like the old version

Reply:
I prefer this format, it makes it look like more like a single instruction as a bulk